### PR TITLE
Link fixes prior to splitting out PX4 v1.13 docs

### PR DESCRIPTION
**Pull request by hamishwillee**

### CODE REVIEW

File: en/sensor_bus/i2c_development.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ The bus is not fast enough even with a single device attached to allow vibration
 
 Drivers should `#include <drivers/device/i2c.h>` and then provide an implementation of the abstract base class `I2C` defined in **I2C.hpp** for the target hardware (i.e. for NuttX [here](https://github.com/PX4/PX4-Autopilot/blob/main/src/lib/drivers/device/nuttx/I2C.hpp)).
 
-A small number of drivers will also need to include headers for their type of device (**drv_*.h**) in [/src/drivers/](https://github.com/PX4/PX4-Autopilot/tree/main/src/drivers) - e.g. [drv_airspeed.h](https://github.com/PX4/PX4-Autopilot/blob/main/src/drivers/drv_airspeed.h).
+A small number of drivers will also need to include headers for their type of device (**drv_*.h**) in [/src/drivers/](https://github.com/PX4/PX4-Autopilot/tree/main/src/drivers) - e.g. [drv_airspeed.h](https://github.com/PX4/PX4-Autopilot/blob/main/src/drivers/drv_airspeed.h).
```
Comment (hamishwillee) [inline en/sensor_bus/i2c_development.md:25]: @bkueng The advice in this line refers to (https://github.com/PX4/PX4-Autopilot/blob/main/src/drivers/drv_airspeed.h , which no longer exists. Is this still relevant? (how do I fix it?)

Comment (hamishwillee) [reply en/sensor_bus/i2c_development.md:25]: PS I'm merging this PR for the other fixes, but if you are able to answer that would be great.

Comment (bkueng) [reply en/sensor_bus/i2c_development.md:25]: You can use another one, like `drivers/drv_led.h`

Comment (hamishwillee) [reply en/sensor_bus/i2c_development.md:25]: Thank you. Fixed in https://github.com/PX4/PX4-user_guide/commit/1c45bdad2f86009118c354ba10866b87a341caa5

**Pull request merged**